src/Numeric/Units/Dimensional/UnitNames/Internal.hs

```diff
@@ -141,12 +141,12 @@ data Prefix = Prefix
                 -- | The name of a metric prefix.
                 prefixName :: PrefixName,
                 -- | The scale factor denoted by a metric prefix.
-                scaleFactor :: Rational
+                scaleExponent :: Int
```

Comment:
Good question.

Options: We could expose `Prefix`, we could expose `Prefix` from an 'Internal' module (actually we may already?).

We could add the IEC/ISO binary prefixes (`kibi`, `mebi`, etc) (which are approved by NIST but not by BIPM). If we wanted to get carried away we could change `Metricality` to have three alternatives instead of two, introduce amount of data as a base dimension, and limit their use only to where ISO/NIST say they should be used.

Outside of units for amount of data (which we currently don't recognize at all), I am not aware of any prefixes that are not powers of ten.